tests/end_to_end/models/aggregator.py

```diff
@@ -49,24 +50,27 @@ def generate_sign_request(self):
         except Exception as e:
             raise ex.CSRGenerationException(f"Failed to generate sign request for {self.name}: {e}")
 
-    def start(self, res_file):
+    def start(self, res_file, run_for_dockerized_ws=False):
```

Comment:
Earlier, we had docker (federation inside container) approach as well, thus suffixed this variable with `dockerized_ws` to avoid confusion. Now that it is removed, a simple name should do.

Renamed to `with_docker` at all relevant places.